### PR TITLE
Resolving or dropping todos related to #93

### PR DESCRIPTION
**Pull request by JackUrb**
# Overview
Picks up some remaining logging TODOs from #93. Overall most of these have been resolved in some form in other places, or are usurped by the broad metrics capabilities introduced by #687.

# Testing
Automated tests

### CODE REVIEW

File: mephisto/abstractions/databases/local_database.py [hunk 1]
```diff
@@ -255,10 +255,6 @@ def init_tables(self) -> None:
         """
         Run all the table creation SQL queries to ensure the expected tables exist
         """
-        # TODO(#93) maybe raise flag when the schema of existing tables isn't what we expect
-        # it to be?
-        # "How to know that schema changes?"
-        # logger.warning("some message")
```
Comment (pringshia) [inline mephisto/abstractions/databases/local_database.py:261]: what happens now in the event of a schema change?

**Review by pringshia — APPROVED** [review]

**Comment by pringshia** [comment]
The version sync test above is failing with `ModuleNotFoundError: No module named 'prometheus_client'` any idea why that may be?

File: mephisto/abstractions/databases/local_database.py (hunk 1 above)
Comment (JackUrb) [reply mephisto/abstractions/databases/local_database.py:261]: We're launching 1.0 and I'm not making any schema changes anymore

**Comment by JackUrb** [comment]
I must not have dependencies quite right on the `prometheus-metrics` branch. Will resolve.

**Comment by pringshia** [comment]
I would expect the Python test to also fail here, interesting that it doesn't

**Comment by JackUrb** [comment]
I think the python test is relying on `requirements.txt` which I updated, but github actions are relying on `pyproject.toml` which I missed.

**Pull request merged**